README.md

```diff
@@ -31,6 +31,12 @@ git --version
 npm -v
 ```
 
+If this is your time building or running a New Relic One application, do the following:  
```

Comment:
I think you meant to say "this is your first time building"

Reply:
Yes I fixed that, thanks for spotting it.  I plan to add more to the readme from a directions perspective so stay tuned for another PR!